src/client/components/AppSidebar/AddCategoryForm.tsx

```diff
@@ -30,7 +30,10 @@ export const AddCategoryForm: React.FC<AddCategoryFormProps> = ({
         maxLength={20}
         placeholder="New category..."
         onChange={(event) => {
-          changeHandler(editingCategoryId, event.target.value)
+          changeHandler(
+            editingCategoryId,
+            event.target.value.charAt(0).toUpperCase() + event.target.value.slice(1)
```

Comment:
You can make this simple by using `lodash`'s `capitalize()`. Also encourage to write unit or e2e test for this.